docs/platforms/javascript/guides/nuxt/index.mdx

```diff
@@ -9,128 +9,90 @@ categories:
-</Alert>
+</Expandable>
 
-## Configure
+## Step 2: Verify Your Setup
 
-Configuration should happen as early as possible in your application's lifecycle.
+If you haven't tested your Sentry configuration yet, let's do it now. You can confirm that Sentry is working properly and sending data to your Sentry project by using the example page created by the installation wizard.
 
-To complete your configuration, add <PlatformLink to="/configuration/options/">options</PlatformLink> to your `Sentry.init()` calls.
+<Expandable level="warning" title="The wizard created an example component instead of a page.">
+The wizard creates a `SentryErrorButton` component when it fails to add an example page to your project. This usually happens if you don't have a `vue.app` file or if that file does not contain the `NuxtPage` component. To verify your setup, follow these steps before continuing with the next section, "View Captured Data in Sentry":
 
-Here, you'll also be able to set context data, which includes data about the <PlatformLink to="/enriching-events/identify-user/">user</PlatformLink>, <PlatformLink to="/enriching-events/tags/">tags</PlatformLink>, or even <PlatformLink to="/enriching-events/context/">arbitrary data</PlatformLink>, all of which will be added to every event sent to Sentry.
+1. Add the `SentryErrorButton` component to a page and open it in your browser. For most Nuxt applications, this will be at localhost.
+2. Click the "Throw Sample Error" button, which triggers an error and starts a performance trace on the client side.
 
-## Verify
+To test the server side as well, refer to the "Verify" section in the [Manual setup guide](/platforms/javascript/guides/nuxt/manual-setup/).
 
-This snippet includes an intentional error, so you can test that everything is working as soon as you set it up.
+</Expandable>
 
-### Server-side
+To test Sentry on the server side, you first need to build your project since Sentry's server-side monitoring **doesn't work in development mode**.\
+Then run your project and make sure to load Sentry on the server side by explicitly adding it via [`--import`](/platforms/javascript/guides/nuxt/install/cli-import/):
 
-Sentry can only be loaded on the server-side by being explicitly added via `--import`.
-Check out the <PlatformLink to="/install/cli-import">`--import` CLI flag</PlatformLink> docs for setup instructions.
+```
+# Start your app after building your project with `nuxi build`
+node --import ./.output/server/sentry.server.config.mjs .output/server/index.mjs
+```
 
-To verify that Sentry works on the server-side, add the following snippet on the server-side:
+After building and running your project:
 
-```js {tabTitle:Nitro} {filename:server/api/sentry-example.get.ts}
-export default defineEventHandler(event => {
-  throw new Error("Sentry Example API Route Error");
-});
-```
+1.  Open the example page `/sentry-example-page` in your browser. For most Nuxt applications, this will be at localhost.
```

Comment:
Should we write the full path here? Since it's always on localhost 🤔 
```suggestion
1.  Open the example page `/sentry-example-page` in your browser. For most Nuxt applications, this will be at localhost:3000/sentry-example-page.
```